lib/framework/NetworkSettingsService.cpp

```diff
@@ -75,6 +75,9 @@ void NetworkSettingsService::manageSTA() {
         });
 
         WiFi.begin(_state.ssid.c_str(), _state.password.c_str()); // attempt to connect to the network
+        #ifdef ARDUINO_LOLIN_C3_MINI
+            WiFi.setTxPower(WIFI_POWER_8_5dBm); //https://www.wemos.cc/en/latest/c3/c3_mini.html#about-wifi
```

Comment:
A few lines above we set ' esp_wifi_set_max_tx_power'  to a selectable value, here it's overwritten by fixed value? Why? The link with` about-wifi` does not work.

Reply:
Haha, they have changed this wiki overnight. They updated the board version now to v2.1, I still have the v1, which has a bug that only works with this setting. It was on the wiki. Seems like they have fixed this with the new version.
https://web.archive.org/web/20220825213136/https://www.wemos.cc/en/latest/tutorials/c3/get_started_with_arduino_c3.html

Reply:
Thanks, now it's clear.

Reply:
Hi @MichaelDvP , I just checked and it does not work with other values, seems to be a hardware issue with this board version.

Reply:
Ok,thanks, will keep an extra pio-env for board v1.0.